docs/src/developer/developer/how-to.md

```diff
@@ -11,21 +11,18 @@ Check that this file exists by running `ls $KUBECONFIG`.
 
 ## How to look at the swagger docs / UI locally
 
-Terminal 1:
-* Set up backing services: `./deploy/dockerephemeral/run.sh`
```

Comment:
Local testings shows that it still needs the backing databases. Maybe this line should be left in, or something to the effect of having DBs accessible.

Reply:
Yeah that's still needed no?

Reply:
oops, you're right.